src/components/Day/HourlyTemperature.js

```diff
@@ -9,9 +9,13 @@ const getTime = (timestamp) => {
     return `${hours}:${minutes}`;
 };
 
+const getTemperature = (temperature) => (
+    Math.round(temperature)
+);
```

Comment:
```suggestion
const getTemperature = (temperature) => Math.round(temperature);
```

Ja pavisam vienkārša f-ja & īsa f-ja, tad var uzreiz return-ot bez iekavām.